lib/index.js

```diff
@@ -1,11 +1,67 @@
+import path from 'path';
+import { readFile } from 'fs';
+import globby from 'globby';
 import postcss from 'postcss';
+import yaml from 'js-yaml';
 import Parser from './parser';
 
 export default postcss.plugin('postcss-map', opts => {
+  opts = opts || {};
+  opts.maps = opts.maps || [];
+  opts.basePath = opts.basePath || process.cwd();
+
+  let maps = Object.create(null);
+  let paths = opts.maps.filter(map => {
+    if (typeof map === 'string') {
+      return true;
+    }
+
+    if (typeof map === 'object') {
+      Object.assign(maps, map);
+    }
+  }).map(map => {
+    return path.resolve(opts.basePath, map);
+  });
+
+  let promise = globby(paths).then(paths => {
```

Comment:
You introduce a new feature: "glob patterns support", which hasn't been discussed before.